packages/components/tests/integration/components/hds/alert/index-test.js

```diff
@@ -18,6 +19,12 @@ module('Integration | Component | hds/alert/index', function (hooks) {
   test('it should render the component with a CSS class that matches the component name', async function (assert) {
     await render(hbs`<Hds::Alert @type="inline" id="test-alert" />`);
     assert.dom('#test-alert').hasClass('hds-alert');
+
+    if (shouldForceAudit()) {
+      // `&enableA11yAudit` set in the URL
+      await a11yAudit();
```

Comment:
It seems like setting the include/exclude in the `test-helpers.js` is not enough, we'd also need them here.

But instead of doing
```js
    await a11yAudit({
      include: [['#ember-testing-container']],
      exclude: [['.flight-sprite-container'], ['.shw-page-main']],
    });
```
in each test file, I'm thinking we can store and export the settings object we currently have in `test-helpers.js` in a variable and import it in each test.

Reply:
That's what I'm working on now, it _should_ be enough to have them in the test helpers file, so I need to get some extra help figuring this out. I must not be doing some syntax correctly.